x-pack/plugins/banners/server/config.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema, TypeOf } from '@kbn/config-schema';
+import { PluginConfigDescriptor } from 'kibana/server';
+import { isHexColor } from './utils';
+
+const configSchema = schema.object({
+  placement: schema.oneOf([schema.literal('disabled'), schema.literal('header')], {
+    defaultValue: 'disabled',
+  }),
+  textContent: schema.string({ defaultValue: '' }),
+  textColor: schema.string({
+    validate: (color) => {
+      if (!isHexColor(color)) {
+        return `must be an hex color`;
+      }
+    },
+    defaultValue: '#000000',
```

Comment:
I might have my designer badge revoked if I pass on an opportunity to suggest hex colors 😄 

My thought is to default to the (computed) EUI colors for a warning callout, so:
- Change the textColor default from #000000 to #8A6A0A
- Change the backgroundColor default from #FFFFFF to #FFF9E8

It's not too in-your-face, it's familiar, and more importantly, it has sufficient color contrast.

<img width="686" alt="Screen Shot 2021-02-05 at 3 50 13 PM" src="https://user-images.githubusercontent.com/446285/107092910-f7e50180-67c9-11eb-8807-87fbe5238628.png">